src/app/components/elements/CareersBanner.tsx

```diff
@@ -86,17 +87,15 @@ export const CareersBanner = () => {
               </CareerCard>
             </Col>
           </Row>
-          <div className="career-comment mt-4">
-            <div className="resources-comment-content d-flex align-items-center">
-              <img src="/assets/star.svg" alt="Star" className="star-img me-3" />
-              <p className="text-left my-3 mx-3 mb-0">
-                &quot;Isaac Computer Science is such a valuable resource for homework and revision. Attending one of
-                their discovery events expanded my understanding of computer science. Since then, I&apos;ve consistently
-                used the platform, finding it invaluable in my computer science journey.&quot; - Lauren Lo
-              </p>
-            </div>
-          </div>
         </Container>
+
+        <TestimonialComment
+          imageSrc="/assets/star.svg"
+          altText="Star"
+          text='"Isaac Computer Science is such a valuable resource for homework and revision. Attending one of
```

Comment:
Similar to what Kris mentioned, could you add this text to a new `content.ts` for the CareersBanner file

Reply:
I don't think that is a good idea because the content.ts file is for the isaac competition only. 
If I was to do this, it would be a content.ts file for a careers. 
That is work for a different ticket as there are many places we can do this.